File: ATC_project/app/src/main/java/com/thesis/atc_project/models/LightIntensity.java
package com.thesis.atc_project.models;

public enum LightIntensity {
    DARK(1),
    SHADY(2),
    MEDIUM(3),
    LIGHT(4),
    BRIGHT(5),
    ;

    private final int value;

    LightIntensity(int i){
        this.value = i;
    }

    public int getValue() {
        return value;
    }
}

File: ATC-AquariumManager/app/src/models/LightIntensity.ts
enum LightIntensity {
  DARK = 1,
  SHADY = 2,
  MEDIUM = 3,
  LIGHT = 4,
  BRIGHT = 5,
}

const LightIntensityToString = (val: number) => {
  switch (val) {
    case LightIntensity.DARK:
      return "Dark";
    case LightIntensity.SHADY:
      return "Shady";
    case LightIntensity.MEDIUM:
      return "Medium";
    case LightIntensity.LIGHT:
      return "Light";
    case LightIntensity.BRIGHT:
      return "Bright";
    default:
      return "";
  }
};

export { LightIntensityToString, LightIntensity };
